src/v3/src/components/AuthCoin/AuthCoin.test.tsx

```diff
@@ -25,10 +25,6 @@ describe('AuthCoin tests', () => {
     };
   });
 
-  afterEach(() => {
-    cleanup();
-  });
```

Comment:
Apparently this happens automatically and has been for a long time. I was the one that added these lines in a PR to reduce test flakiness but perhaps it wasn't the issue and didn't help (nor did it harm) our tests.

https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-cleanup